kodilla-stream/src/main/java/com/kodilla/optional/homework/Application.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.optional.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class Application {
+
+    public static void main(String[] args) {
+
+        List<Student> students = new ArrayList<>();
+        students.add(new Student("Meghan", new Teacher("John")));
+        students.add(new Student("Danny", null));
+        students.add(new Student("Liz", null));
+        students.add(new Student("Robert", new Teacher("Mike")));
+
+        for (Student student : students) {
+            System.out.println("Student: " + student.getName() + ", " + "Teacher: " + undefinedTeacher(student));
+        }
+    }
+
+    public static String undefinedTeacher(Student student) {
```

Comment:
Co to za nazwa metody?
W ogole ta metida nie jest potrzebna. Tworzysz tu jakis obiekt, nie wiedzac, czy go bedziesz musiL
al uzyc czy nie